<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SrcRecCdpDistributionDialog</class>
 <widget class="QDialog" name="SrcRecCdpDistributionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>646</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Source/Receiver/CDP Distribution</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>862</width>
        <height>626</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="closeBtn">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>99</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="zoomInBtn">
       <property name="text">
        <string>Zoom In</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomOutBtn">
       <property name="text">
        <string>Zoom Out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fit2winBtn">
       <property name="text">
        <string>Fit to window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomInXBtn">
       <property name="text">
        <string>Zoom In X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomInYBtn">
       <property name="text">
        <string>Zoom In Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomOutXBtn">
       <property name="text">
        <string>Zoom Out X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomOutYBtn">
       <property name="text">
        <string>Zoom Out Y</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="srcCheckBox">
       <property name="text">
        <string>Shot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="recCheckBox">
       <property name="text">
        <string>Receiver</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cdpCheckBox">
       <property name="text">
        <string>CDP</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
